### ISSUE TITLE
Peertube channels not getting updated

### ISSUE THREAD

**Issue by Thann** [LemmyNet/lemmy#2480]
### Issue Summary
The peertube channels I follow on lemmy are not seeing new posts unless I search for them.
See: https://lemmy.ml/c/podcast@h3h3.club
doesn't show the last 4 videos from: https://h3h3.club/c/podcast/videos

### Steps to Reproduce

1. follow peertube channel from lemmy
2. wait for new post to be made to channel 
3. see that lemmy view of channel doesnt have new post.

### Technical details
basically a regression of #2331 
initially resubscribing made the first two videos show up, but got nothing after that and resubbing again does nothing.

<3

**Comment by Nutomic** [comment]
At this time both links you posted appear to be perfectly in sync. I also checked the logs, and there are many activities coming in from that channel, including Update activities which seem to be processed without any errors. Additionally I was able to follow the channel from [this test instance](https://voyager.lemmy.ml/c/podcast@h3h3.club) without any errors.

So all in all, i cant reproduce any problem here.

**Comment by Thann** [comment]
hmm, thats odd, because I waited for a week for them to sync before posting the bug :thinking:

**Comment by Nutomic** [comment]
Lemmy also refetches communities and its posts (or in this case, channels and videos) when they are viewed in the browser, maybe thats what happened. Though its weird that you wouldnt have triggered it before.

**Comment by Thann** [comment]
Maybe the initial sync from a user subbing works, but after that it doesnt?
Perhaps, it has something to do with my user, because It's not updating again. 


![Screenshot from 2022-10-07 09-16-25](https://user-images.githubusercontent.com/578515/194600499-cfffb32b-ab79-4b68-942c-87acabe654ac.png)

**Comment by Nutomic** [comment]
Hmm could it be that the Peertube instance or that channel is broken? There is nothing logged for the last videos in Lemmy logs, and they are not even listed in the outbox. Can you reproduce the problem with other Peertube instances?

**Comment by ravermeister** [comment]
I've got the exact same behaviour for e.g. news@mp-tube.de on my instance

**Comment by Nutomic** [comment]
I tried following the mentioned channels from ds9.lemmy.ml which runs the dev version of lemmy, and it seems to work fine there (https://ds9.lemmy.ml/c/news@mp-tube.de, https://ds9.lemmy.ml/c/podcast@h3h3.club). So this should be fixed in 0.17 when that gets released (you might have to refollow Peertube channels then). 

This could be because 0.16 uses a newer version of http signatures, which doesnt seem to work properly with Peertube. Dev version switches to an older version for compatibility.

**Comment by dessalines** [comment]
Since that's in the main branch then, we can consider this fixed, and re-open if there's problems after the 0.17 release.
